django/views/debug.py

```diff
@@ -101,6 +101,9 @@ class ExceptionReporterFilter:
     contain lenient default behaviors.
     """
 
+    def get_safe_request_meta(self, request):
```

Comment:
yeah I saw that there weren't existing tests so I wasn't sure if it was needed. Will do, but it will be tomorrow, thanks for all the help tim :+1: